mapviz_plugins/src/textured_marker_plugin.cpp

```diff
@@ -71,8 +71,13 @@ namespace mapviz_plugins
     p3.setColor(QPalette::Text, Qt::red);
     ui_.status->setPalette(p3);
 
+    // Initialize the alpha value to default
+    alphaVal = 1.0f;
```

Comment:
If this is meant as a default value for this member variable, it'd be better for it to be part of the initializer list at the top of the constructor.